### DISCUSSION TITLE
Week 8

### DISCUSSION THREAD

**Discussion by Stev3eM** [SE-TINF22B6/CookHub#70] [Show and tell]
Hello backpackers,

This is our weekly discussion-blog to keep you up to date within our project progression.
Week 8 has approched and we got some new updates for you:
<br><br>

### User Guide
-----
✅ We proudly present you our new team member Chef Carlos! He will represent our team on the website and gives guidance and advices to new and old customers. (created with Stable Diffusion)<br>

![Chef Carlos](https://github.com/SE-TINF22B6/CookHub/blob/main/Frontend/src/assets/Chef_Carlo_with_background.png?raw=true)

###  Landing Page
------
✅ We finally implemented the first mockup design (Landing page) to the project. (of course with Carlos)

![image](https://github.com/SE-TINF22B6/CookHub/assets/113697513/35706f19-682c-44d2-8122-39d3a6af617c)

### Login Page
-----
✅ We also implemented the login page mockup design (with Carlos first advice)

![image](https://github.com/SE-TINF22B6/CookHub/assets/113697513/59da79a5-5387-4fcc-9e1e-6a1867984152)

### Topic
-----
✅ We updated our [srs.md](https://github.com/SE-TINF22B6/CookHub/blob/main/Project_Doc/srs.md#3-nonfunctional-requirements) to include our Architecture Significant Requirements (ASR)

✅ We clarified our quality attribute requirements using a utility tree & chose the 3 most relevant scenarios for our product development

![image](https://github.com/SE-TINF22B6/CookHub/assets/115070676/265f85d0-f123-4d38-95e7-e1260036a7bd)

### OpenAI ChatGPT 
-----
✅ **Research & tests of more specific prompts to get better resolutions with [adventurize it Ⓒ](https://github.com/SE-TINF22B6/CookHub/wiki/adventurize-it-%E2%92%B8)** 

<br><br>

See you next time!

**Your**   <img width="50" alt="image" src="https://github.com/SE-TINF22B6/CookHub/assets/113697513/1a218c37-d3b0-4c88-9ae1-827b5a4eba42">   **Team**

**Comment by AndresMasis** [comment]
Hi

First of all. I loved chef Carlos. It is super nice.

Second, the landing website has a nice appearance, I like the UI. 

Also I consider that the non functional requirements distributions you made are very accurate for this project. I consider very important the emphasis you did in the usability.

Congratulations for the solid progress. Keep it like that.

Cheers, CookPit.

**Comment by ins305** [comment]
Hi Team CookHub,

I really like your idea of having Chef Carlos, who guides your users around the website. It's a unique feature of your website and makes it special.

I also think you did a really good job at implementing your UI design. One thing I noticed is that some of the text is quite hard to read because of the strong shadow if you are a bit further away (but that's just complaining on a high level).

You also have a good variety of important quality attribute scenarios. The top 3 seem to be a good choice, although I'm having a bit of trouble with understanding what exacly you mean with "the user should have no problems in using the website". Does it mean that the website is intuitive or that the user doesn't encounter any bugs? Maybe you can clarify this scenario.

Overall, you're making really good progress on your project and we're looking forward to your future development

**Comment by TadaXD20** [comment]
Hi CookHub,

I love your mascot. It will make it more fun for the customer to explore your website and makes the experience unique.
I understand what you want to accomplish with your goals but I think your 3 top goals are to general. Specify them more e.g. with help of the SMART criterium. It will make it easier for you to measure if you reached these goals. Also I would like to know why you chose these 3 exact goals. Maybe add an explanation with advantages or an excerpt of your discussion.

I'm looking forward to your next achievements
Tanina from Team Plapy
